src/js/_enqueues/lib/emoji-loader.js

```diff
@@ -422,17 +420,8 @@ new Promise( ( resolve ) => {
 			settings.supports.everythingExceptFlag &&
 			! settings.supports.flag;
 
-		// Sets DOMReady to false and assigns a ready function to settings.
-		settings.DOMReady = false;
-		settings.readyCallback = () => {
-			settings.DOMReady = true;
-		};
```

Comment:
This code appears to be a vestige of when the emoji was first introduced in 4.2:

https://github.com/WordPress/wordpress-develop/blob/7e57307b7539d100e23932295c4fa1018de7eea0/src/wp-includes/js/wp-emoji-loader.js#L65-L85

The logic was greatly simplified since then, now that `DOMContentLoaded` is supported by all supported browsers. Also, now that this script is loaded as a module, the DOM will have already been loaded anyway. So it seems like this logic and the corresponding logic in `src/js/_enqueues/wp/emoji.js` is now unnecessary.